packages/fuels-core/src/code_gen/abigen.rs

```diff
@@ -1,7 +1,9 @@
 use std::collections::HashMap;
 
 use crate::code_gen::bindings::ContractBindings;
-use crate::constants::{ADDRESS_SWAY_NATIVE_TYPE, CONTRACT_ID_SWAY_NATIVE_TYPE};
+use crate::constants::{
+    ADDRESS_NATIVE_TYPE, CONTRACT_ID_NATIVE_TYPE, OPTION_NATIVE_TYPE, RESULT_NATIVE_TYPE,
```

Comment:
I don't find much benefit in constants that are likely to be used in only one place. Inlining them helps with clarity and if the time comes to change them, you only change them at the one place they're used.